Makefile

```diff
@@ -1,22 +1,61 @@
+ARCH    ?= amd64
+ALL_ARCH = amd64 arm arm64 ppc64le s390x
+
 .PHONY: all test
 
 IMAGE_BUILD_CMD := docker build
 
 VERSION := $(shell git describe --tags --dirty --always)
 
+IMAGEARCH ?=
+QEMUARCH  ?=
+
+ifeq ($(ARCH),amd64)
+        IMAGEARCH =
+        QEMUARCH  = x86_64
+else ifeq ($(ARCH),arm)
+        IMAGEARCH = arm32v7/
+        QEMUARCH  = arm
+else ifeq ($(ARCH),arm64)
+        IMAGEARCH = arm64v8/
+        QEMUARCH  = aarch64
+else ifeq ($(ARCH),ppc64le)
+        IMAGEARCH = ppc64le/
+        QEMUARCH  = ppc64le
+else ifeq ($(ARCH),s390x)
+        IMAGEARCH = s390x/
+        QEMUARCH  = s390x
+else
+	$(error unknown arch "$(ARCH)")
+endif
+
 IMAGE_REGISTRY := quay.io/kubernetes_incubator
 IMAGE_NAME := node-feature-discovery
 IMAGE_TAG_NAME := $(VERSION)
 IMAGE_REPO := $(IMAGE_REGISTRY)/$(IMAGE_NAME)
 IMAGE_TAG := $(IMAGE_REPO):$(IMAGE_TAG_NAME)
 
+ifneq ($(ARCH),amd64)
+	IMAGE_TAG = $(IMAGE_REPO)-$(ARCH):$(IMAGE_TAG_NAME)
```

Comment:
Tab indentation should be changed to spaces

Reply:
Done.